ui/src/components/layout/Markdown.vue

```diff
@@ -185,6 +185,9 @@
             }
         }
     }
+    p {
```

Comment:
Hi @riyamustare, could you please make this as a separate class and not just a property on `p`, as like this, it can make unwanted changes in other part of the project?

Reply:
Also, if you could add `5px` padding, I think it would look much better.

Reply:
sure ill work on it